addon/templates/components/polaris-data-table/cell.hbs

```diff
@@ -0,0 +1,72 @@
+{{#if presentational}}
+  {{#if header}}
+    <th aria-hidden="true" role="presentation" class={{cellClassNames}} style={{style}}></th>
+  {{else}}
+    <td aria-hidden="true" role="presentation" class={{cellClassNames}} style={{style}}></td>
+  {{/if}}
+{{else}}
+  {{#if (or header fixed)}}
+    {{#if header}}
+      {{#if sortable}}
+        <th
+          scope="col"
+          class={{cellClassNames}}
+          style={{style}}
+          role="button"
+          aria-sort={{sortDirection}}
+          aria-label={{sortAccessibilityLabel}}
+          tabindex="0"
+          {{action onSort}}
```

Comment:
`onclick` will pass an event object to the action while this usage doesn't - for the keydown handler we need that event to determine whether it was the Enter key that was pressed, whilst for the click handler we don't need it. I prefer to not pass that object if possible since it's something of an implementation detail and shouldn't really be exposed to the outside world IMO.